starter-code/public/scripts/article.js

```diff
@@ -93,20 +93,30 @@ Article.all = rawData.map(function(ele){
   /*Article.allAuthors = () => {
     return Article.all.map().reduce();
   };*/
-
   Article.numWordsByAuthor = () => {
     return Article.allAuthors().map(author => {
-      // TODO: Transform each author string into an object with properties for
+      var art = Article.all.filter(function(item){
+        return item.author === author
+         }).map(function(item){
+            return item.body.split(' ').length;
+             }).reduce(function(start,next){
+                return start + next;
+                });
+      return {
+        authorName: author
+        ,authorWords: art
```

Comment:
@bbeeksma probably best to get in the habit of aligning your comma placement with JS convention - this still feels a bit weird to me.

For diff purposes, do feel free to include a trailing comma on the last elements in arrays/objects:

```javascript
return {
  authorName: author,
  authorWords: art,
};
```